docs/resources/ocean_aks_2.0.md

```diff
@@ -0,0 +1,244 @@
+    min_vcpu = 2
+    max_vcpu = 16
+    min_memory_gib = 10
+    max_memory_gib = 18
+    architectures = ["X86_64"]
+    series = ["D v3"]
+  }
+  
+  // ----------------------------------------------------------------------------
+  
+  // ----------------------------------------------------------------------------
+}
+```
+
+```
+output "ocean_id" {
+  value = spotinst_ocean_aks_np.example.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `aks` - (Required) AKS cluster configuration. Cannot be updated.
+  * `aks_cluster_name` - (Required) The name of the AKS Cluster.
+  * `aks_infrastructure_resource_group_name` - (Required) The name of the cluster's infrastructure resource group.
+  * `aks_region` - (Required) The cluster's region.
+  * `aks_resource_group_name` - (Required) The name of the cluster's resource group.
+* `autoscaler` - (Optional) The Ocean Kubernetes Autoscaler object.
+    * `autoscale_is_enabled` - (Optional, Default: `true`) Enable the Ocean Kubernetes Autoscaler.
+    * `autoscale_down` - (Optional) Auto Scaling scale down operations.
+        * `max_scale_down_percentage` - (Optional) The maximum percentage allowed to scale down in a single scaling action.
+    * `resource_limits` - (Optional) Optionally set upper and lower bounds on the resource usage of the cluster.
+        * `max_vcpu` - (Optional) The maximum cpu in vCpu units that can be allocated to the cluster.
+        * `max_memory_gib` - (Optional) The maximum memory in GiB units that can be allocated to the cluster.
+    * `autoscale_headroom` - (Optional) Spare resource capacity management enabling fast assignment of pods without waiting for new resources to launch.
+        * `automatic` - (Optional) Automatic headroom configuration.
+            * `percentage` - (Optional) Optionally set a number between 0-100 to control the percentage of total cluster resources dedicated to headroom.
+* `controller_cluster_id` - (Required) Enter a unique Ocean cluster identifier. Cannot be updated.
+* `health` - (Optional) The Ocean AKS Health object.
+    * `grace_period` - (Optional, Default: `600`) The amount of time to wait, in seconds, from the moment the instance has launched until monitoring of its health checks begins.
+* `name` - (Required) Add a name for the Ocean cluster.
+* `scheduling` - (Optional) An object used to specify times when the cluster will turn off. Once the shutdown time will be over, the cluster will return to its previous state.
+    * `shutdown_hours` - (Optional) An object used to specify times that the nodes in the cluster will be taken down.
+        * `is_enabled` - (Optional) Flag to enable or disable the shutdown hours mechanism. When False, the mechanism is deactivated, and the cluster remains in its current state.
+        * `time_windows` - (Optional) The times that the shutdown hours will apply.
+* `headrooms` - (Optional) Specify the custom headroom per VNG. Provide a list of headroom objects.
```

Comment:
it says per VNG instead of cluster, get clarification on openAPI document then change accordingly